frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -15,6 +15,9 @@ import { getLegend } from '../MapLegend/MapLegend'
 
 import './DownloadAnalysisLink.css'
 
+const useStaticCsvs: boolean = process.env.ENV === 'PRD'
+const staticCsvRootUrl: string = 'https://s3-us-west-2.amazonaws.com/encompass-public-data/results-csv/'
```

Comment:
This could be extracted if/as/when we want to use different S3 paths for different environments or somesuch.